src/Fixer/Phpdoc/PhpdocReturnSelfReferenceFixer.php

```diff
@@ -157,7 +157,7 @@ protected function createConfigurationDefinition(): FixerConfigurationResolverIn
                         if (!isset($default[$from])) {
                             throw new InvalidOptionsException(sprintf(
                                 'Unknown key "%s", expected any of "%s".',
-                                \is_object($from) ? \get_class($from) : \gettype($from).(\is_resource($from) ? '' : '#'.$from),
```

Comment:
cannot form be object/resource?

Reply:
No, it cannot, see line 152, it's a key of an array.